scripts/allowlist/genesisContent/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
did you make separate packages for a reason? would it be simpler for managing dependencies etc to combine them?

Reply:
Combining them is probably a good idea ...